frontend/src/store.ts

```diff
@@ -11,7 +11,7 @@ export const store = configureStore({
     stepper: stepperReducer,
     config: configReducer,
     jiraVerifyResponse: jiraVerifyResponseReducer,
-    pipelineToolVerifyResponse: pipelineToolResponseReducer,
+    pipelineToolVerifyResponse: pipelineToolVerifyResponseReducer,
     sourceControlVerifyResponse: sourceControlVerifyResponseReducer,
```

Comment:
could the 3 xxxResponseReducer to merged into one ResponseReducer ?

Reply:
Yep, i think that we can fix that later about refactor